ios/reactNativeApp.xcodeproj/project.pbxproj

```diff
@@ -773,6 +773,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
+				CODE_SIGN_IDENTITY = "iPhone Developer";
```

Comment:
Why this change?

cc @ferrannp?

Reply:
This was from Luke. I don't know. I can remove that?